lib/dwylbot/endpoint.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Dwylbot.Endpoint do
+  use Phoenix.Endpoint, otp_app: :dwylbot
+
+  socket "/socket", Dwylbot.UserSocket
+
+  # Serve at "/" the static files from "priv/static" directory.
+  #
+  # You should set gzip to true if you are running phoenix.digest
+  # when deploying your static files in production.
+  plug Plug.Static,
+    at: "/", from: :dwylbot, gzip: false,
+    only: ~w(css fonts images js favicon.ico robots.txt)
+
+  # Code reloading can be explicitly enabled under the
+  # :code_reloader configuration of your endpoint.
+  if code_reloading? do
+    socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
+    plug Phoenix.LiveReloader
+    plug Phoenix.CodeReloader
+  end
+
+  plug Plug.RequestId
+  plug Plug.Logger
+
+  plug Plug.Parsers,
+    parsers: [:urlencoded, :multipart, :json],
+    pass: ["*/*"],
+    json_decoder: Poison
+
+  plug Plug.MethodOverride
+  plug Plug.Head
+
+  # The session will be stored in the cookie and signed,
+  # this means its contents can be read but not tampered with.
+  # Set :encryption_salt if you would also like to encrypt it.
+  plug Plug.Session,
+    store: :cookie,
+    key: "_dwylbot_key",
+    signing_salt: "Guv2WEhg"
```

Comment:
After a bit more research on the signing_salt I don't think it is necessary to put it in an environment variable, see also the similar  issue on Phoenix https://github.com/phoenixframework/phoenix/issues/2146
The signing_salt is combined with the secret_key_base which is private.
I've tried to add the signing_salt in an environment variable but this break my tests so I think I will leave it clear for now as the dwylbot doesn't create any cookie and I prefer focusing on the logic of #7